src/Elmish.WPF/ViewModel.fs

```diff
@@ -358,10 +362,11 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
 
   let bindings =
     log "[%s] Initializing bindings" propNameChain
-    let dict = Dictionary<string, VmBinding<'model, 'msg>>()
-    for b in bindings do
+    let dict = Dictionary<string, VmBinding<'model, 'msg>>(bindings.Length)
```

Comment:
I initialized the dictionary using the [constructor that accepts a capacity](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.-ctor?view=netframework-4.8#System_Collections_Generic_Dictionary_2__ctor_System_Int32_).  This is (slightly) more efficient since we know that number is exactly how many elements will be in the dictionary when we are done adding.  This is unrelated to everything else in this PR.

Reply:
👍